### ISSUE TITLE
CI builds should have version -ci-build

### ISSUE THREAD

**Issue by dt-r** [hl7au/au-fhir-core#97]
The current version of this master should be 0.3.0-ci-build? or 0.3.0-draft? Personally I think it is ci-build as draft still implies a snapshot not a CI build that is changing constantly.

All versions that aren't the final release should have a qualifier in the version number that indicates release status. 0.3.0-draft or 0.3.0-ballot or 0.3.0-ci-build. No qualifier indicates it is the final.

The HL7 AU versioning and release criteria need to be published on Confluence and AU Core should match in it's master and in it's release branches.

**Comment by dbojicic** [comment]
Added qualifier -ci-build. 
@brettesler-ext will document versioning and release criteria as part of the release documentation.

**Comment by dt-r** [comment]
@dbojicic - is this now documented and we can close this off?

**Comment by dbojicic** [comment]
Yes, this is documented in https://confluence.hl7.org/spaces/HAFWG/pages/273942690/HL7+AU+artefact+identification+naming+versioning+and+publication+policies#HL7AUartefactidentification,naming,versioning,andpublicationpolicies-Versioningpolicy.
